README.md

```diff
@@ -211,3 +234,7 @@ tidb_vs.delete(["f8e7dee2-63b6-42f1-8b60-2d46710c1971"])
 # delete with filter
 tidb_vs.delete(["f8e7dee2-63b6-42f1-8b60-2d46710c1971"], filter={"category": "P1"})
 ```
+
+## Examples
```

Comment:
It would be better to introduce examples with a simple but attractive description.